frappe/public/js/frappe/widgets/widget_dialog.js

```diff
@@ -237,9 +237,19 @@ class ShortcutDialog extends WidgetDialog {
 				hidden: 1,
 			},
 			{
-				fieldtype: "Color",
+				fieldtype: "Select",
 				fieldname: "color",
 				label: __("Color"),
+				options: ["Grey", "Green", "Red", "Orange", "Pink", "Yellow", "Blue", "Cyan", "Teal"],
+				default: "Grey",
+				onchange: () => {
+					let color = this.dialog.fields_dict.color.value.toLowerCase();
+					let $select = this.dialog.fields_dict.color.$input;
+					if (!$select.parent().find('.color-box').get(0)) {
+						$(`<div class="color-box"></div>`).insertBefore($select.get(0));
+					}
+					$select.parent().find('.color-box').get(0).style.backgroundColor = color;
```

Comment:
Use css variables for background color, since we are using different color hash for indicators.
```suggestion
					$select.parent().find('.color-box').get(0).style.backgroundColor = `var(--bg-${color})`;
```

Reply:
Color is very light cannot see it clearly
![image](https://user-images.githubusercontent.com/30859809/116695642-82f72180-a9de-11eb-83d6-6673b21f825c.png)

Reply:
right, you can use `var(--text-on-${color})` instead for darker color. (we also use this for indicator dot)